aries-site/src/pages/templates/popover.mdx

```diff
@@ -0,0 +1,67 @@
+import { Example } from '../../layouts';
+import {
+  PopoverAnatomy,
+  PopoverSimpleExample,
+  PopoverInlineExample,
+  PopoverDoExample,
+  PopoverDontPlacementExample,
+  PopoverDontNestExample
+} from '../../examples';
+
+<Example
+  figma="https://www.figma.com/design/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=34141-2"
+  width="100%"
+  code='https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/popover/PopoverSimpleExample.js'
+>
+  <PopoverSimpleExample />
+</Example>
+
+## Use cases
+### Providing inline contextual help
+
+A Popover can be triggered through an icon-only button to offer helpful contextual information in a non-intrusive manner.
+
+<Example
+  figma="https://www.figma.com/design/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=34141-2"
+  width="100%"
+  code='https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/popover/PopoverInlineExample.js'
+>
+  <PopoverInlineExample />
+</Example>
+
+## Anatomy
+
+<Example caption="Diagram illustrating the 4 parts of a Popover." plain>
+  <PopoverAnatomy />
+</Example>
+
+| Label | Region        | Purpose                              | Required | Notes                                      |
+| :---: | ------------- | ------------------------------------ | :------: | ------------------------------------------ |
+| **1** | **Title**     | Identifies the Popover.               | Optional |                                            |
+| **2** | **Close Button** | Closes the Popover.                | Required |                                            |
+| **3** | **Body**      | Provides contextual information.      | Required | May include non-text elements.             |
+| **4** | **Footer**    | Contains additional actions.          | Optional | May include anchors.                       |
+
+
+## Content guidelines
```

Comment:
Suggestion to follow the same way of writing for all bullet points here: 

Popover's content should be:
- **Relevant:** It should relate to its trigger.
- **Not crucial:** It should not include crucial information or actions needed to complete a task.
- **Short:** It should be concise to avoid overwhelming the user and taking up excessive screen space. Follow [voice and tone](https://design-system.hpe.design/foundation/voice-and-tone) guidelines for writing effective content.

Reply:
+1 looks like we lost the "bullet" points

Reply:
added bullets back!